recipes/docloud/meta.yaml

```diff
@@ -0,0 +1,49 @@
+# This is the recipe for DOcplexcloud Python Client
+
+{% set version = "1.0.155" %}
+{% set md5 = "8ab30ce2790ac8842f088c089b2308de" %}
+
+package:
+  name: docloud
+  version: {{ version }}
+
+source:
+  url: https://pypi.python.org/packages/df/6d/26adee77b13f9fc7c761a6e2c97af79d49a33a2832341bbbdb34933f7fe6/docloud-{{ version }}.tar.gz
+  md5: 8ab30ce2790ac8842f088c089b2308de
+
+build:
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    # When setuptools is available add the `--single-version-externally-managed --record record.txt` above.
+    - setuptools
+  run:
+    - python
+    - requests
+    - six
+    - enum34  # [not py3k]
+
+test:
+  imports:
+     - docloud
+
+about:
+  home: https://onboarding-oaas.docloud.ibmcloud.com/software/analytics/docloud/
+  # Remember to specify the license variants for BSD, Apache, GPL, and LGLP.
+  # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
+  # See https://opensource.org/licenses/alphabetical
+  license: Apache-2.0
```

Comment:
Hmm...might not be one in the current tarball from PyPI. Could it please be added to `MANIFEST.in` in the source? That way it can be included in the tarball on PyPI and easily included here.